pkg/handler/sign/handler.go

```diff
@@ -6,9 +6,46 @@ import (
 	"net/http"
 	"sort"
 
+	"github.com/miekg/pkcs11"
+	"github.com/open-component-model/signing-server/pkg/crypto11"
 	"github.com/open-component-model/signing-server/pkg/encoding"
 )
 
+/*
+type HSMContext struct {
+	Ctx     *pkcs11.Ctx
+	Session pkcs11.SessionHandle
+	Priv    pkcs11.ObjectHandle
+}
+
+func NewHSMContext(ctx *pkcs11.Ctx, session pkcs11.SessionHandle, priv pkcs11.ObjectHandle) *HSMContext {
+	return &HSMContext{
+		Ctx:     ctx,
+		Session: session,
+		Priv:    syscall.PR_FP_EXC_OVF,
+	}
+}
+*/
```

Comment:
Is this comment here for a reason?

Reply:
This was for the old implementation without the wrapper.

I'll remove it,